test/gtest/uct/ib/test_cq_moderation.cc

```diff
@@ -12,13 +12,12 @@ extern "C" {
 #include <poll.h>
 #include <infiniband/verbs.h>
 
-static const unsigned nsec_per_usec = (UCS_NSEC_PER_SEC / UCS_USEC_PER_SEC);
-
-/* wait for 3 usecs to get statistics */
+/* wait for 3 secs to get statistics */
 static const unsigned long test_period = (3ul * UCS_USEC_PER_SEC);
 static const unsigned moderation_period = 1000; /* usecs */
-static const unsigned event_limit = (40 * test_period / moderation_period / nsec_per_usec);
-static const unsigned max_repeats = 1000;
+/* use multiplier 2 because we have same iface to send/recv which may produce 2x events */
+static const unsigned event_limit = (2 * test_period / moderation_period);
```

Comment:
1. let's add "_usec" suffix to test_period and moderation_period , to avoid confusion about units
2. can you pls add UCS_TEST_MESSAGE to print number of actual events and event limit? eight before checking events<event_limit

Reply:
added suffix & test message